src/main.ts

```diff
@@ -126,10 +126,10 @@ async function pack(name: string, options: any) {
     try {
         winston.info("Webpacking project");
         await WebpackRunner.run({
+            ignoredModules: config.ignoredModules,
```

Comment:
For these linting updates, could you update tslint to reflect them? My linting tools don't capture these as issues.